weekly_report/generate_report.py

```diff
@@ -16,6 +15,7 @@
 import altair as alt
 import pandas as pd
 import boto3
+from dotenv import load_dotenv
```

Comment:
Good attention to detail by adding `dotenv` library, due to different approach with ECS task compared to Lambda